pkg/streamingpromql/testdata/ours/aggregators.test

```diff
@@ -625,3 +700,20 @@ eval range from 0 to 12m step 6m count_values by (idx) ("idx", series)
 # Once that fix is vendored into Mimir, we can remove the two (\\")? groups below.
 eval_fail instant at 0 count_values("a\xc5z", series)
   expected_fail_regexp invalid label name "(\\")?a\\(\\)?xc5z(\\")?"( for count_values)?
+
+clear
+
+load 6m
+  series{idx="1"}   1 2 3 4 5 6
+  series{idx="2"}   4 5 6 7 8 9
+  series{idx="3"}   7 8 Inf NaN -Inf
+
+# Quantile value warning is emitted even when no series are returned
+eval_warn range from 0 to 12m step 6m quantile(20, noseries)
+
+eval_warn range from 0 to 12m step 6m quantile(Inf, noseries)
+
+eval_warn range from 0 to 12m step 6m quantile(-Inf, noseries)
+
+eval range from 0 to 30m step 6m quantile(0.9, series)
```

Comment:
Could you add a test like this with `quantile(0, series)` and `quantile(1, series)` to confirm whether Inf and NaN values sort to the start or end?